source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,547 @@
+# One-time Sync Company Custom 
+
+This section lists all endpoints that must be implemented for a one-time synchronous company custom check integration.
+
+One-time synchronous company custom checks are custom checks with the following behaviour:
+
+    * The check result will be returned directly upon request; the response will not be sent until the check is finished (synchronous)
+    
+    * The check operates at a point-in-time: there is no ongoing monitoring of the check result (one-time)
+
+
+## Get Check Configuration (One-time Sync Custom Company)
+
+<%= partial("includes/partials/config.md.erb", :locals => { :name => "sync_company_custom" }) %>
+
+This endpoint specifies the configuration used by PassFort for performing
+one-time synchronous custom checks using your integration.
```

Comment:
Change to singular (apparently better for translations when we do them)

"...performing a one-time synchronous custom check using..."